Userland/Libraries/LibWeb/Layout/Node.cpp

```diff
@@ -77,6 +77,9 @@ bool Node::establishes_stacking_context() const
     auto position = computed_values().position();
     if (position == CSS::Position::Absolute || position == CSS::Position::Relative || position == CSS::Position::Fixed || position == CSS::Position::Sticky)
         return true;
+    auto opacity = computed_values().opacity();
+    if (opacity.has_value() && opacity.value() != 1.0f)
```

Comment:
I think there's a `Optional<T>::operator==(T)`, could also use that here

Reply:
Using that operator crashes the page

Reply:
Oh, this is `!=`, not `==`...
my bad, sorry.

Reply:
It does check for existence, you just end up with `!has_value() || value() != 1.0f` instead of what you currently have :P